projects/lib-shared-modules/src/lib/components/dialog-popup/dialog-popup.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="flex flex-row justify-between items-center bg-primary h-10">
+    <mat-label mat-dialog-title class="text-white">{{dialogueData?.header}}</mat-label>
+    <img class="cursor-pointer pr-2" [mat-dialog-close]="false" src="./../assets/icons/cross-icon.png" alt="">
+</div>
+<div mat-dialog-content class="text-secondary p-4">
+    <h3>{{dialogueData?.content}}</h3>
+</div>
+<div class="border-t"></div> 
+<div mat-dialog-actions class="flex flex-row justify-end items-center space-x-4 py-0">
+  <button  class="btn text-primary" [mat-dialog-close]="false">{{dialogueData.cancelButton |translate}}</button>
```

Comment:
same here, please explain "[mat-dialog-close]="false"

Reply:
this is the inbuilt function from angular-dialog module, so instead of writing separate function to close. using this function to close.